scripts/deploy-gh-pages.js

```diff
@@ -3,27 +3,38 @@ var ghpages = require('gh-pages');
 var basePath = path.join(__dirname, '../dist');
 var repoUrl = require('../package.json').repository.url;
 
-var makeConfig = function(token) {
-  if (token) {
+var GH_TOKEN = process.env.GH_TOKEN;
+var tokenRegex = GH_TOKEN ? new RegExp(GH_TOKEN, 'g') : null;
+
+var log = function(message) {
+  if (tokenRegex) {
+    // Hide token from build logs
+    message = message.replace(tokenRegex, '[GH_TOKEN]');
+  }
+
+  console.log(message);
+};
+
+var makeConfig = function() {
```

Comment:
I've been deliberately optimising for readable config without forcing you to execute the code in your head, hence why I'm maintaining two separate objects. Does that sound okay?